org.osgi.test.cases.cm/bnd.bnd

```diff
@@ -30,19 +30,23 @@ Import-Package: ${-signaturetest}, *
 	
 -signaturetest                      = org.osgi.service.cm
 
--buildpath = \
-    org.osgi.test.support;version=project, \
-    org.osgi.framework;maven-scope=provided;version=1.8,\
-    org.osgi.resource;maven-scope=provided;version=1.0,\
-    org.osgi.service.permissionadmin;maven-scope=provided;version=1.2,\
-    org.osgi.namespace.implementation;version=1.0,\
-    org.osgi.namespace.service;version=1.0,\
-    org.osgi.service.coordinator;version=1.0,\
-	org.osgi.service.cm;version=latest
+-buildpath: \
+	org.osgi.test.support;version=project,\
+	org.osgi.framework;maven-scope=provided;version='1.8',\
+	org.osgi.resource;maven-scope=provided;version='1.0',\
+	org.osgi.service.permissionadmin;maven-scope=provided;version='1.2',\
+	org.osgi.namespace.implementation;version='1.0',\
+	org.osgi.namespace.service;version='1.0',\
+	org.osgi.service.coordinator;version='1.0',\
+	org.osgi.service.cm;version=latest,\
+	org.osgi.annotation.versioning,\
```

Comment:
Why does the TCK need versioning annotations? And it does need them, it should use `osgi.annotations;version=8.0`.